Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/SharedLogic/FarePolicy.hs

```diff
@@ -178,12 +188,23 @@ mkFarePolicyBreakups mkValue mkBreakupItem mbDistance mbTollCharges farePolicy =
       nightShiftEndItem,
       nightShiftStartInSecondsItem,
       nightShiftEndInSecondsItem,
-      congestionChargePercentageItem
+      congestionChargePercentageItem,
+      insuranceChargeItem,
+      cardChargePercentageItem,
+      fixedCardChargeItem
     ]
     <> additionalDetailsBreakups
     <> driverExtraFeeBoundsMinFeeItems
     <> driverExtraFeeBoundsMaxFeeItems
   where
+    mkInsuranceChargeCaption = \case
+      Meter -> show Tags.INSURANCE_CHARGE_PER_METER
```

Comment:
Why do we want to create so many tags? We can just convert the value to Meters and just send into meters to BAP.

Reply:
In UI wouldn't it be better to have it separate. 
Like lets suppose we configure fare policy in `Meters` in one city and `Miles` in another (using distanceUnit column).
We will communicate same to BAP which will be used by rider-app-ui as well in same way so will be consistent across.